test-fixture.html

```diff
@@ -146,6 +146,8 @@
 (function () {
   var TestFixturePrototype = Object.create(HTMLElement.prototype);
   var TestFixtureExtension = {
+    constructor: HTMLElement,
```

Comment:
shouldn't this be `TestFixture`? I'm slightly confused by the manner in which the TestFixture constructor and prototype are defined here though.